etc/schema.json

```diff
@@ -5507,27 +5507,39 @@
                             "type": "object",
                             "properties": {
                                 "flows_checked": {
+                                    "description":
+                                            "number of flows checked for timeout in the last pass",
                                     "type": "integer"
                                 },
                                 "flows_evicted": {
+                                    "description": "number of flows that were evicted or timed out",
```

Comment:
what is the difference between evicted and timed out?

Reply:
Well, both are timed out.. But, from what I can tell,
1. evicted == flows that were not owned by the thread and were added to the `evicted` list of the flow bucket. <- so set up for a later pick up by the `Flow Manager`
2. timed out == flows that were a part of the normal `head` list of the flow bucket so owned by FM but have timed out

Both ultimately seem to be added to the aside queue.

With the explanation above, I'm realizing that these details do not matter to the user so saying `evicted or timed out` is incorrect and confusing. 😅